### DISCUSSION TITLE
noPublicForm dans la transmission des formulaires Web

### DISCUSSION THREAD

**Discussion by IhsaneRQ** [MTESSDev/FRW#160] [Annonces]
Bonjour, 

Est-ce qu'on peut avoir le noPublicForm parmi les données transmises lors de votre appel à notre API pour transmettre les formulaires Web? Cela va nous permettre de faire le lien avec le formulaire à mettre à jour dans notre BD, parce que actuellement vous envoyez juste le noFormulaire et noConfirmation que nous n'avons nul part au moment de la création du formulaire Web.
Aussi, c'est quoi la différence entre noFormulaire et noPublicForm ?

Merci et bonne fin de journée

**Comment by anisite** [comment]
Bonsoir :) 

Effectivement vous soulevez une incohérence, je crois qu'il n'était pas prévu que le noFormulaire se retrouve en sortie, il s'agit du numéro équivalent à la clée primaire dans notre BD c'est un numéro séquentiel, tandis que le noPublicForm lui est en réalité un GUID aléatoire, il est converti pour être raccourcis.

Nous n'avons pas de problèmes à ajouter l'autre identifiant, vous avez besoin de celà le plus tôt possible j'imagine?

**Reply by IhsaneRQ** [reply]
Bonjour, 

Merci d'abord pour votre retour rapide. 

Effectivement, nous avons besoin de cela le plus tôt possible svp.

Merci beaucoup et bonne journée

**Reply by anisite** [reply]
Bon après-midi, le correctif est déployé dans l'environnement "IT" bons tests !

**Reply by IhsaneRQ** [reply]
Bon après-midi, 

Merci beaucoup pour votre collaboration, c'est très apprécié.

Bonne fin de semaine
